container/docker/handler.go

```diff
@@ -31,6 +31,8 @@ import (
 	dockerutil "github.com/google/cadvisor/utils/docker"
 	"github.com/google/cadvisor/zfs"
 
+	"strconv"
```

Comment:
nit: we usually place all of the golang imports (e.g. fmt, time, strconv) together at the top.